plugin/pkg/admission/exec/admission_test.go

```diff
@@ -106,7 +105,7 @@ func TestAdmission(t *testing.T) {
 	}
 }
 
-func testAdmission(t *testing.T, pod *api.Pod, handler *DenyExec, shouldAccept bool) {
+func testAdmission(t *testing.T, pod *corev1.Pod, handler *DenyExec, shouldAccept bool) {
```

Comment:
is this change correct? Isn't this the pod being admitted?

Reply:
No, it isn't. we don't put any obj into `CONNECT` attribute